pages/techblog/[id]/index.page.tsx

```diff
@@ -51,17 +53,28 @@ export default function Page() {
 
       case 'success':
         const { company } = data;
+        const handleCareerClick = () => {
+          const newWindow = window.open('', '_blank');
+          if (newWindow) {
+            newWindow.location.href = company.careerUrl;
+          }
+        };
         return (
           <article className='px-[20.4rem] py-[6.4rem]'>
             <>
               <TechDetailCard {...data} />
               <section className='flex items-center justify-between px-[3.2rem] py-[3.1rem] border border-gray2 rounded-[1.6rem]'>
                 <CompanyTitle
-                  title={company?.name}
+                  title={company.name}
                   content='는 절찬리 채용중! 확인하러
               가볼까요?'
                 />
-                <MainButton text='채용정보 보러가기' variant='primary' icon={<HandRight />} />
+                <MainButton
+                  text='채용정보 보러가기'
+                  variant='primary'
+                  icon={<HandRight />}
+                  onClick={handleCareerClick}
+                />
```

Comment:
onClick으로 안하고 next/Link로 더 편하게 할 수 있어용!
```suggestion
           <Link href={company.careerUrl} target='_blank'>
                <MainButton
                  text='채용정보 보러가기'
                  variant='primary'
                  icon={<HandRight />}
                />
            </Link>  
```

Reply:
[649a1d6](https://github.com/dreamyPatisiel/devdevdev-client/pull/34/commits/649a1d650ccccedd60ca138dcc120a3509c96ca9) 😊